README.md

```diff
@@ -11,14 +11,20 @@ Download the `beavis-ci.sh` script and put it on your path, e.g.:
 curl -o ~/bin/beavis-ci.sh https://raw.githubusercontent.com/LSSTDESC/beavis-ci/master/beavis-ci.sh
```

Comment:
I don't think this would work. The script needs to access the svg files in the badges folder. Probably just ask people to clone this repo?

Reply:
I thought of that: the script now downloads the badges it needs when it starts up. What we _should_ do is package this properly so that you can `brew upgrade` etc, but I don't know how to do that for shell scripts.